wls-infomanagement-service/src/main/resources/db/migrations/h2/V0_2__dropTableTheEntity.sql

```diff
@@ -0,0 +1 @@
+-- DROP TABLE theEntity;
```

Comment:
warum ist das auskommentiert?

Reply:
copy paste fehler - mal wieder 😶 

siehe commit [36a1d70](https://github.com/it-at-m/Wahllokalsystem/pull/440/commits/36a1d70f79b3e42e319db09b4eb930921129135b)